shared/common-adapters/popup-menu.desktop.js

```diff
@@ -51,6 +53,10 @@ class Menu extends Component<void, Props, void> {
   }
 }
 
+PopupMenu.defaultProps = {
+  visible: true,
```

Comment:
lets actually remove the visible flag. i think the better animation things just wrap the component so it'd be a HOC thing instead of the component doing it itself.